apis/core/bootstrap/shapes/cube-project.ts

```diff
@@ -46,6 +46,8 @@ ${shape('cube-project/create')} {
       ${sh.maxCount} 1 ;
       ${sh.nodeKind} ${sh.IRI} ;
       ${sh.order} 40 ;
+      ${dash.hidden} true ;
+      ${sh.defaultValue} <https://lindas.admin.ch/foen/cube> ;
```

Comment:
Just wondering if it would not be a good thing to have this in a env variable?

Reply:
Well ackchully, I think this would fit most in some kind of system-wide configuration, which we don't have.

The difference being that env requires redeployment, and ideally this would not be necessary for this.

Reply:
I agree, but still I think that it would be useful to at least be able to have different values for different instances